actions/event_calendar/addtocalendar.php

```diff
@@ -16,12 +16,16 @@
 		
 	if (event_calendar_add_personal_event($event_guid, $user_guid)) {
 		remove_entity_relationship($user_guid, 'event_calendar_request', $event_guid);
-		notify_user($user_guid, elgg_get_site_entity()->guid, elgg_echo('event_calendar:add_users_notify:subject'),
-			elgg_echo('event_calendar:add_users_notify:body', array(
-			$user->name,
-			$event->title,
-			$event->getURL()
-		)));
+		if ($user_guid != elgg_get_logged_in_user_guid()) {
+			$user_language = ($user->language) ? $user->language : (($site_language = elgg_get_config('language')) ? $site_language : 'en');
```

Comment:
Yes, I think it works. (This is maybe a bit overkill because 95% of community plugins do not check this at all. But it doesn't hurt to be safe I guess.)